src/avt/Database/Ghost/avtUnstructuredDomainBoundaries.C

```diff
@@ -296,20 +536,25 @@ avtUnstructuredDomainBoundaries::GetGivenIndex(int from, int to)
 //
 //  Programmer:  Akira Haddox
 //  Creation:    August 11, 2003
+// 
+//  Modifications:
+//    Justin Privitera, Wed Apr 23 17:39:24 PDT 2025
+//    Style updates.
+//    Added const where possible.
 //
 // ****************************************************************************
 
 template <class T>
 void
-CopyPointer(T *src, T *dest, int components,
-                                                              int count)
+CopyPointer(T *src, T *dest, int components, int count)
 {
-    int i;
-    int nIter = count * components;
+    const int nIter = count * components;
 
     *dest = *src;
-    for (i = 1; i < nIter; ++i)
+    for (int i = 1; i < nIter; ++i)
```

Comment:
Why not `std::memmove(dst, src, nIter)`?

Reply:
I think we can just use `std::copy` and delete this function entirely.

Reply:
Done.